examples/python/compression/client.py

```diff
@@ -21,9 +21,8 @@
 import logging
 
 import grpc
-
-from examples.protos import helloworld_pb2
-from examples.protos import helloworld_pb2_grpc
+import helloworld_pb2
```

Comment:
I'm wondering how this works under Bazel. PYTHONPATH should be pointing to the repo root in that case. How are we picking up `helloworld_pb2` there?

I'm all for enabling running the example under both Bazel and non-Bazel, but we have to enable that deterministically.

Reply:
Looks like in the Bazel test we're using `subprocess` to start client and server, so technically those scripts was started  at parent directory?
https://github.com/grpc/grpc/blob/352c33986e5069abbdc8496915d8f913230c15df/examples/python/compression/test/compression_example_test.py#L22-L24

I think that's why we have `examples/python/compression` added to PYTHONPATH.

Reply:
Resolved offline. This actually seems to be behavior provided by Bazel to emulate a script being run from its own directory. Surprising, but harmless.